src/commands/lambda/__tests__/mock-configs/no-functions.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Do you still need this file since `fs.readFile` is mocked?

Reply:
sorry, thought I deleted them, fixed